### PR TITLE
#24: [First] Dockerize Leader Service

### PR DESCRIPTION
**Pull request by owenxr**
### Major Changes:
- Reconfigure Kafka for multiple containers in the same network
- Implement python code to connect first service to leader runner

Issue: #24

### CODE REVIEW

File: first/main.py [hunk 1]
```diff
@@ -16,6 +17,8 @@ def start():
     branch = request.form['branch']
 
     worker_id = "0" # TODO: Replace
```
Comment (jakejack13) [inline first/main.py:19]: This should be replaced with the id of the Docker container running the worker

File: first/utils/docker_runner.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"""Utility module for interacting with the docker daemon"""
+
+import subprocess
+
+def run_docker_container(repo, branch, num_workers, img_url):
```
Comment (jakejack13) [inline first/utils/docker_runner.py:5]: Add documentation to this function as well as types on both parameters and return value

**Review by jakejack13 — CHANGES_REQUESTED** [review]
Couple changes before merge

**Review by jakejack13 — APPROVED** [review]
LGTM

**Review by MitchellGray100 — APPROVED** [review]

**Pull request merged**